### PR TITLE
Add explicit mention of beta server to login screen of beta product flavour.

### PR DESCRIPTION
**Pull request by mithlesh4257**
Fix to issue #1058.
Add explicit mention of beta server to login screen of beta product flavour.

issue1058

@misaochan , @psh , @neslihanturan, kindly review.

### CODE REVIEW

**Comment by psh** [comment]
I built the app using the "production" debug product flavor expecting the message to be hidden, but it still appears.

broken-login

Looking at the code, the layout changes were made in the `src/main` folder - that means they are global across all build types and product flavors.  If you keep the layout global like that, you'll need to add code that's sensitive to the _variant_ to specifically hide the message when the app is pointing at the production environment.

**Comment by mithlesh4257** [comment]
Thanks @psh :-), I'm fixing it.

**Comment by neslihanturan** [comment]
Thanks for your contribution @mithlesh4257 .Maybe you can search for "how to recognize build variant at runtime" there is a lot of solutions like this https://stackoverflow.com/questions/23431354/how-to-get-the-build-variant-at-runtime-in-android-studio . Still, if it is hard for you, you can leave this task for someone else to handle.

Besides, can you please be more specific on your commit messages? Instead of "Update file" you can write "Create x method to do x" etc.

**Comment by mithlesh4257** [comment]
Thanks @neslihanturan :)

**Comment by mithlesh4257** [comment]
https://stackoverflow.com/questions/36569134/how-to-hide-or-add-menu-items-for-flavor
https://stackoverflow.com/questions/21221868/exclude-specific-build-variants
https://stackoverflow.com/questions/23431354/how-to-get-the-build-variant-at-runtime-in-android-studio 
I have gone through all three, I have one doubt, Is there possible directly hide a TextView during runtime?
One more doubt, 
flavours

see above, we have two flavors prod and beta, but in "app/src" folder, according to my reading in stacksoverflow I get to know that what the names of flavor that should also be as directory in "app/src".
Help @neslihanturan , @maskaravivek, @neslihanturan, @psh.

**Comment by neslihanturan** [comment]
Same problem also here, there is unnecessary code changes under this PR.

File: app/src/main/java/fr/free/nrw/commons/upload/SingleUploadFragment.java [hunk 1]
```diff
@@ -50,7 +50,7 @@
     @BindView(R.id.titleEdit) EditText titleEdit;
     @BindView(R.id.descEdit) EditText descEdit;
     @BindView(R.id.titleDescButton) Button titleDescButton;
-    @BindView(R.id.share_license_summary) TextView licenseSummaryView;
+    @BindView(R.id.share_license_summary) HtmlTextView licenseSummaryView;
```
Comment (neslihanturan) [inline app/src/main/java/fr/free/nrw/commons/upload/SingleUploadFragment.java]: No need to this edits under this PR, irrelevant.

File: app/src/main/res/layout/fragment_single_upload.xml [hunk 2]
```diff
@@ -50,7 +50,7 @@
         android:id="@+id/titleDescButton"
         />
 
-    <TextView
+    <fr.free.nrw.commons.ui.widget.HtmlTextView
```
Comment (neslihanturan) [inline app/src/main/res/layout/fragment_single_upload.xml]: No need to this edits under this PR, irrelevant.

**Comment by neslihanturan** [comment]
I think it is better if you leave this PR for now, and focus on other PRs of you @mithlesh4257 . There is a lot to fix also here.

**Pull request closed without merging**